crates/polars-plan/src/plans/aexpr/function_expr/log.rs

```diff
@@ -10,8 +10,21 @@ pub(super) fn entropy(s: &Column, base: f64, normalize: bool) -> PolarsResult<Co
     }
 }
 
-pub(super) fn log(s: &Column, base: f64) -> PolarsResult<Column> {
-    Ok(s.as_materialized_series().log(base).into())
+pub(super) fn log(columns: &[Column]) -> PolarsResult<Column> {
+    assert_eq!(columns.len(), 2);
+
+    let s = &columns[0].as_materialized_series();
+    let base = &columns[1];
+
+    polars_ensure!(
+        base.len() == 1,
+        ComputeError: "base must be a single value."
```

Comment:
not sure i'm misunderstanding the feature request (i could well be!) but i think the request is to be able to do something like
```
df = pl.DataFrame({'a': [1,2,3], 'b': [4,5,6]})
df.select(pl.col('a').log(pl.col('b')))
```
where `base` is the same length as `s`, but and each row can get `log`-ed with a different base?